apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessPreviewList.tsx

```diff
@@ -0,0 +1,145 @@
+import { useIntl } from "react-intl";
+
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { PreviewList, PreviewMetaData, Well } from "@gc-digital-talent/ui";
+
+import { getClassificationName } from "~/utils/poolUtils";
+import {
+  getQualifiedRecruitmentStatusChip,
+  isQualifiedFinalDecision,
+} from "~/utils/poolCandidate";
+
+import { RecruitmentDate } from "./MetadataDate";
+import ReviewRecruitmentProcessDialog from "./ReviewRecruitmentProcessDialog";
+
+const ReviewRecruitmentProcessPreviewList_Fragment = graphql(/* GraphQL */ `
+  fragment ReviewRecruitmentProcessPreviewList on PoolCandidate {
+    ...ReviewRecruitmentProcessDialog
+    id
+    finalDecisionAt
+    suspendedAt
+    removedAt
+    placedAt
+    finalDecision {
+      value
+    }
+    pool {
+      id
+      name {
+        localized
+      }
+      classification {
+        group
+        level
+        minSalary
+        maxSalary
+      }
+    }
+  }
+`);
+
+interface ReviewRecruitmentProcessPreviewListProps {
+  recruitmentProcessesQuery: FragmentType<
+    typeof ReviewRecruitmentProcessPreviewList_Fragment
+  >[];
+}
+
+const ReviewRecruitmentProcessPreviewList = ({
+  recruitmentProcessesQuery,
+}: ReviewRecruitmentProcessPreviewListProps) => {
+  const intl = useIntl();
+
+  const recruitmentProcesses = getFragment(
+    ReviewRecruitmentProcessPreviewList_Fragment,
+    recruitmentProcessesQuery,
+  ).filter(
+    (recruitmentProcess) =>
+      recruitmentProcess.finalDecisionAt &&
+      isQualifiedFinalDecision(recruitmentProcess.finalDecision?.value),
```

Comment:
see: https://github.com/GCTC-NTGC/gc-digital-talent/actions/runs/13074982240/job/36484825551#step:11:91

Reply:
Fixed in [078eac4](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/078eac46db2fce25a7adb6689e87d4d264cade38)